### PR TITLE
Improve dummy data in SQLI emulator database

### LINKED ISSUE #131: Sqli dummy data
We can use this library for making our dummy data. In this case, this data will look more genuine.
https://github.com/lk-geimfari/elizabeth

### PR DESCRIPTION
**Pull request by rnehra01**
Fix #131

### CODE REVIEW

File: tanner/utils/db_helper.py [hunk 1]
```diff
@@ -54,14 +48,13 @@ def insert_dummy_data(table_name, data_tokens, cursor):
...
-                    data = random.choice(dummy_data) + "@" + str.lower(random.choice(dummy_data)) \
-                           + random.choice(domains)
+                    data = elizabeth.Personal().email()
                     values.append(data)
                 if token == 'P':
-                    data = random.choice(dummy_data)
+                    data = elizabeth.Personal().password()
                     values.append(data)
                 if token == 'T':
                     data = 'This is a comment number' + str(i)
```
Comment (afeena) [inline tanner/utils/db_helper.py]: Can we use elizabeth `Text()` here?

File: tanner/utils/db_helper.py [hunk 2]
```diff
@@ -54,14 +48,13 @@ def insert_dummy_data(table_name, data_tokens, cursor):
                 if token == 'I':
                     values.append(i)
                 if token == 'L':
-                    data = random.choice(dummy_data)
+                    data = elizabeth.Personal().full_name()
```
Comment (afeena) [inline tanner/utils/db_helper.py]: L is for login/username data, maybe `elizabeth.Personal().username()` more suitable her?

**Comment by rnehra01** [comment]
@afeena Is it okay?

**Comment by afeena** [comment]
@rnehra01 👍

**Pull request merged**